src/labelle/gui/q_settings_toolbar.py

```diff
@@ -57,7 +57,7 @@ def _init_elements(self) -> None:
 
     def update_labeler_context(
         self,
-        supported_tape_sizes: tuple[int, ...],
+        supported_tape_sizes: list[int],
```

Comment:
Is this really needed? In case the list is constant, it would be better as a convention to prefer tuple over list

Reply:
The amount of supported tape sizes varies from printer to printer. I got an error that I tried to change the "tuple" size. A quick google search gave this as a solution. If there is a better solution please elaborate.

Reply:
Ah, tuples are "immutable" so you should always generate a new one instead of modifying an old one.

Reply:
Fixed (It didn´t complain this time)